src/slack_block_util.ts

```diff
@@ -249,6 +247,17 @@ export const voterStatusPanel: SlackBlock = {
   ],
 };
 
+export function loadingSlackView(): SlackView {
+  return {
+    title: {
+      type: 'plain_text',
+      text: 'Loading...',
+    },
+    blocks: [],
```

Comment:
We should probably have a single section block with some loading text in it. I think Slack does funny things when there are no blocks at all.